README.md

```diff
@@ -386,6 +386,8 @@ Split.configure do |config|
   # before experiment reset or deleted
   config.on_before_experiment_reset  = -> (example) { # Do something on reset }
   config.on_before_experiment_delete = -> (experiment) { # Do something else on delete }
+  # after experiment winner had been set
+  config.on_winner_choose = -> (experiment) { # Do something on winner choose }
```

Comment:
What do you think about renaming it to `on_experiment_winner_choose` ? (keeping all experiment hooks with the same naming pattern)

Reply:
Ok, will change.

Reply:
Hey @andrehjr, I have updated a hook name.

Sorry for delay 😅

Reply:
Awesome, no worries 💖 Thanks for taking the time!